flaml/searcher/blendsearch.py

```diff
@@ -624,9 +687,11 @@ def suggest(self, trial_id: str) -> Optional[Dict]:
         # Number of threads is 1 or 2. Thread 0 is a vacuous thread
         assert len(self._search_thread_pool) < 3, len(self._search_thread_pool)
         if len(self._search_thread_pool) < 2:
-            # When a local converges, the number of threads is 1
+            # When a local thread converges, the number of threads is 1
             # Need to restart
             self._init_used = False
+            if self._candidate_start_points is not None and self._points_to_evaluate:

```

Comment:
Assuming self._points_to_evaluate will be suggested first as long as they exist? Need to add comments or move this to suggest.